#rrbot:
#
#  # Publish all joint states -----------------------------------
#  joint_state_controller:
#    type: joint_state_controller/JointStateController
#    publish_rate: 50  
#  
#  # Controlador implementado en python ---------------------------------------
#  right_wheel_controller:
#    type: my_controller/MyPositionController
#    joint: right_wheel_hinge
#    gain: 0.1 # sin uso
#  
#  left_wheel_controller:
#    type: my_controller/MyPositionController
#    joint: left_wheel_hinge
#    gain: 0.1


rrbot:

  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Controlador implementado en python ---------------------------------------
  right_wheel_controller:
    type: effort_controllers/JointVelocityController
    joint: right_wheel_hinge
    pid: {p: 100.0, i: 0.01, d: 10.0}
    
  
  left_wheel_controller:
    type: effort_controllers/JointVelocityController
    joint: left_wheel_hinge
    pid: {p: 100.0, i: 0.01, d: 10.0}
